test/client/sign.ts

```diff
@@ -177,7 +177,7 @@ describe('client.sign', function () {
 
     assert.throws(() => {
       this.client.sign(request.txJSON, secret)
-    }, /1\.2 is an illegal amount/)
+    }, /1\.2 is an illegal amount/u)
```

Comment:
Why is there a `u` added to the end?

Reply:
I added it to fix a linter issue earlier. But, now linter seems to be fine without the `u` as well

Reply:
Try refreshing the page - the linter should complain about that

Reply:
Omar - to answer your question: https://eslint.org/docs/rules/require-unicode-regexp